features/manage-position/Create.tsx

```diff
@@ -12,9 +12,7 @@ import Position from "../../containers/Position";
 import PriceFeed from "../../containers/PriceFeed";
 import Etherscan from "../../containers/Etherscan";
 
-const Container = styled(Box)`
-  max-width: 720px;
-`;
+const Container = styled(Box)``;
```

Comment:
Why not just replace `Container` with `Box` if you're gonna remove CSS overrides anyways?

Reply:
yup. good point. I will go through all these containers and replace with Box and remove the redundant CSS.